### PR TITLE
responsive landing on mobile

### PR DESCRIPTION
**Pull request by aldriebarcena**
Screen Shot 2023-11-13 at 3 42 27 PM
Screen Shot 2023-11-13 at 3 42 43 PM

### CODE REVIEW

File: src/components/about/Landing.jsx [hunk 1]
```diff
@@ -5,19 +5,19 @@ const Landing = () => {
...
             <Image src={chart} className="h-full w-auto m-0" alt="chart" />
           </div>
           <div className="px-2 flex w-full justify-center items-start">
             <div className="flex flex-col w-3/4 lg:w-auto">
-              <p className="text-5xl py-1.5 font-bold text-transparent bg-clip-text bg-gradient-to-r from-hiss-blue to-hiss-purple">
+              <p className="text-5xl text-center py-1.5 font-bold text-transparent bg-clip-text bg-gradient-to-r from-hiss-blue to-hiss-purple">
                 Highlander Statistics Society
               </p>
-              <p className="mt-0 my-0 text-white px-0 py-2.5 font-bold text-center text-3xl h-14 w-4/5 bg-gradient-to-r from-hiss-blue to-hiss-purple rounded-full">
+              <p className="sm:m-0 mr-24 text-white px-0 py-2.5 font-bold text-center sm:text-3xl sm:h-14 sm:w-4/5 w-80 bg-gradient-to-r from-hiss-blue to-hiss-purple rounded-full">
```
Comment (SafeDuck) [inline src/components/about/Landing.jsx:17]: is hard coding large values like mr-24 and w-80 necessary?

File: src/components/about/Landing.jsx [hunk 2]
```diff
@@ -5,19 +5,19 @@ const Landing = () => {
...
                 at Unversity of California Riverside
               </p>
-              <p className="my-1 py-1 h-10 w-6/12">
+              <p className="sm:my-1 py-1 mb-10 h-10 sm:w-6/12 w-full">
```
Comment (SafeDuck) [inline src/components/about/Landing.jsx:20]: same thing with the mb-10

Is you screenshot showing on desktop or laptop?

Comment (aldriebarcena) [reply src/components/about/Landing.jsx:20]: showing on desktop monitor

File: src/components/about/Landing.jsx (hunk 1 above)
Comment (aldriebarcena) [reply src/components/about/Landing.jsx:17]: it was so the shape can be longer and centered on mobile

**Review by SafeDuck — APPROVED** [review]

**Pull request merged**